cpp/include/cudf/io/parquet_metadata.hpp

```diff
@@ -89,11 +73,18 @@ struct parquet_column_schema {
   [[nodiscard]] auto name() const { return _name; }
 
   /**
-   * @brief Returns parquet type of the column.
+   * @brief Returns parquet physical type of the column.
+   *
+   * @return Column parquet physical type
+   */
+  [[nodiscard]] auto type() const { return _type; }
+
+  /**
+   * @brief Returns parquet physical type of the column.
    *
-   * @return Column parquet type
+   * @deprecated in branch-25.06 and to be removed in a future release; Use `type()` instead.
    */
-  [[nodiscard]] auto type_kind() const { return _type_kind; }
+  [[deprecated]] [[nodiscard]] auto type_kind() const { return _type; }
```

Comment:
Deprecating instead of renaming to avoid breaking change.

Reply:
isn't it returning a different type now?

Reply:
The same thing.